pkg/network/http/gotls/lookup/main.go

```diff
@@ -9,4 +9,4 @@ package lookup
 // - Use /var/tmp/datadog-agent/system-probe/go-toolchains
 //   as the location for the Go toolchains to be downloaded to.
 //   Each toolchain version is around 500 MiB on disk.
-//go:generate go run ./internal/generate_luts.go --test-program ./internal/program.go --package lookup --out ./luts.go --min-go 1.13 --arch amd64,arm64 --shared-build-dir /var/tmp/datadog-agent/system-probe/go-toolchains
+//go:generate go run -tags linux_bpf ./internal/generate_luts.go --test-program ./internal/program.go --package lookup --tags linux_bpf --out ./luts.go --min-go 1.13 --arch amd64,arm64 --shared-build-dir /var/tmp/datadog-agent/system-probe/go-toolchains
```

Comment:
Is the `-tags linux_bpf` (the single dash version) needed here?

Reply:
yes because it will not be able to compile pkg/network/go/bininspect/

Reply:
yeah it's not right, it's seems the issue bpf_linux tags dependency in pkg/network/go and shouldn't been set in these directort/subdirs